bundles/org.openhab.binding.samsungtv/README.md

```diff
@@ -86,3 +87,18 @@ String  TV_KeyCode       "Key Code"                            (gLivingRoomTV)
 Switch  TV_Power         "Power"                               (gLivingRoomTV)   { channel="samsungtv:tv:livingroom:power" }
 Switch  TV_ArtMode       "Art Mode"                            (gLivingRoomTV)   { channel="samsungtv:tv:livingroom:artMode" }
 ```
+
+### Apps
+
+List of known apps and the respectve name that can be passed on to the `sourceApp` channel. Values are confirme to work on UE50MU6179.
```

Comment:
```suggestion
List of known apps and the respective name that can be passed on to the `sourceApp` channel.
Values are confirme to work on UE50MU6179.
```

Use a new line for each sentence

Reply:
Looks like I can't do any changes to what was submitted or proposed, So if you don't mind, kindly fix the typo in "Values are confirme to work on UE50MU6179." for me as well. Might save us from starting over with a new PR. Sorry I'm new to Github.

Reply:
done